src/content/docs/kubernetes-pixie/kubernetes-integration/get-started/kubernetes-components.mdx

```diff
@@ -0,0 +1,129 @@
+---
+title: New Relic Kubernetes components
+tags:
+  - Integrations
+  - Kubernetes integration
+  - Kubernetes components
+metaDescription: "Learn what components are deployed after installing Kubernetes."
+---
+
+import kubernetesv3Integration from 'images/kubernetes_diagram_v3-integration.png'
+
+
+After deploying Kubernetes you have a cluster, which consists of a set of worker machines, called nodes, that run containerized applications. Every cluster has at least one worker node. Kubernetes runs your workload by placing containers into pods to run on nodes.
+
+The control plane is the container orchestration layer that exposes the API and interfaces to define, deploy, and manage the lifecycle of containers. The control plane manages also the worker nodes and the pods in the cluster.
+
+There're four distinct components that need to be monitored in a Kubernetes environment each with their specificities and challenges:
+
+* Infrastructure
+* Containers
+* Applications
+* Kubernetes clusters
```

Comment:
I am unsure about the choice of words and/or what we want to convey here. Can I get some more context about what purpose is this section serving?
I ask because I do not recognize these four distinct components as something that is official Kubernetes nomenclature, or something the industry tends to use. Kubernetes often refers as "[Kubernetes components](https://kubernetes.io/docs/concepts/overview/components/)" to the list of pieces that form a Kubernetes cluster, which is not what we are defining here.

Moreover I have some doubts about this categorization as well, e.g. what do we understand by "Infrastructure" in the first point. It would seem to me that "Infrastructure" is a very high level and wide group, that would contain "Kubernetes clusters". In turn, "Kubernetes clusters" is also wide and would include "Containers".

If you provide some clarification on what do we want to explain in this section I can help with some terminology or wording that is more familiar to a Kubernetes user.

Reply:
I don't think the solution was to remove it completely, rather that you clarify what information you were trying to convey here so that we could help you with the technical terminology.